### PR TITLE
SIG Apps Annual report 2023

### PR DESCRIPTION
**Pull request by soltysh**
Fixes https://github.com/kubernetes/community/issues/7763

/cc @janetkuo @kow3ns 
/assign @mrbobbytables

/hold
for approvals

### CODE REVIEW

File: sig-apps/annual-report-2023.md [hunk 1]
```diff
@@ -42,11 +34,9 @@
     - [2804 - Consolidate Workload controllers life cycle status](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2804-consolidate-workload-controllers-status) - v1.27
     - [2879 - Track ready Pods in Job status](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2879-ready-pods-job-status) - v1.29
     - [3140 - TimeZone support in CronJob](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3140-TimeZone-support-in-CronJob) - v1.27
-    - [3715 - Elastic Indexed Job](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3715-elastic-indexed-job) - v1.28
```
Comment (kannon92) [inline sig-apps/annual-report-2023.md:37]:
```suggestion
 - [4026 - Job Creation TimeStamp ](https://github.com/kubernetes/enhancements/issues/4026)
```

**Review by janetkuo — APPROVED** [review]

File: sig-apps/annual-report-2023.md (hunk 1 above)
Comment (soltysh) [reply sig-apps/annual-report-2023.md:37]: This one's included in beta section.

**Review by mrbobbytables — COMMENTED** [review]
/approve
👍

**Pull request merged**